src/order/bounded_lattice.lean

```diff
@@ -730,6 +730,43 @@ lemma lt_iff_exists_coe_btwn [partial_order α] [densely_ordered α] [no_top_ord
 ⟨λ h, let ⟨y, hy⟩ := dense h, ⟨x, hx⟩ := (lt_iff_exists_coe _ _).1 hy.2 in ⟨x, hx.1 ▸ hy⟩,
  λ ⟨x, hx⟩, lt_trans hx.1 hx.2⟩
 
+/- Lemma shows that a in with_top Z is either the maximal element or some integer n. -/
```

Comment:
```suggestion
/-- Lemma shows that a in with_top Z is either the maximal element or some integer n. -/
```
If you use the `/--` then the comment becomes a "docstring" and will show up in the mouse-over tooltips in VScode.